.github/workflows/mirror-develop.yaml

```
@@ -0,0 +1,21 @@
+name: Push develop to GitLab
+
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  sync-develop:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout full history
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0  # Full history
+
+      - name: Push develop to GitLab
+        # GAT must be in format https://oauth2:<GROUP_ACCESS_TOKEN>@tools.ages.pucrs.com/username/repo.git
```

Update comment to new format
```suggestion
        # GAT is in format https://oauth2:<GROUP_ACCESS_TOKEN>@tools.ages.pucrs.br/wimbelemdon-mais
```

---

De acordo. Alterado localmente.